cfpb/cfgov-refresh
# Replacing static form with use_form convention & Changes to office page

Starting the process of removing the static content from the FOIA pages that were created in the last sprint. A few changes to the office page to better deal when there's no content for a section. Added a way to add forms to office pages when the "use forms" box is checked that will likely evolve in the next upcoming sprints.

## Additions

- Added `forms/` to the includes
- Added logic to include forms on sub-page template

## Removals

- Goodbye to the static FOIA pages made pre-template.

## Changes

- Moved `submit-a-request` form to includes
- Changed some office template logic to better detect blank sections.

## Testing

- Visit http://localhost:7000/offices/privacy/. That page used to have a blank section and doesn't anymore!
- Visit http://localhost:7000/sub-pages/submit-a-request/ to see a form

## Review

- @sebworks 
- @anselmbradford 
- @jimmynotjim 

## Preview

[Preview this PR without the whitespace changes](?w=0)

## Notes

- Talked with @sebworks about forms and using this an approach for this sprint while he works on macros to streamline and make creating forms easier in the future.
- There's a few indenting changes I didn't make on the template files `_single.html` since multiple people are working on them and I want to reduce merge conflicts until they're more stable.
